server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -123,7 +129,12 @@ public Repo<Manager> call(long tid, Manager manager) throws Exception {
           Text currentRow = null;
           int dirCount = 0;
 
-          boolean sawTabletAvailability = false;
+          TabletAvailability initialAvailability = tableInfo.initialAvailability;
+          if (initialAvailability == null) {
+            log.error("Initial tablet availability is null and shouldn't be, defaulting to "
+                + TabletAvailability.ONDEMAND);
+            initialAvailability = TabletAvailability.ONDEMAND;
+          }
```

Comment:
It seems like this should only be null if there is a bug in the code, if that is the case could possibly do the following.

```suggestion
          TabletAvailability initialAvailability = Objects.requireNonnull(tableInfo.initialAvailability);
```